initial-data-load/README.md

```diff
@@ -0,0 +1,27 @@
+| NODE_ENV                           | no       | development                                                        | This environment variable is used to state in our case whether a particular environment is development, production, test or provision environment. Express uses it to alter its own default behavior. Fo example a common use-case is running additional debugging or logging code in a development environment. |
+| PORT                               | no       | 8080                                                               | The port on which the service is running.                                                                                                                                                                                                                                                                        |
+| LOG_LABEL                          | no       | oscar-initial-data-upload                                          | Title of the logs that you can see in the terminal when you run the service.                                                                                                                                                                                                                                     |
+| AZURE_STORAGE_CONNECTION_STRING    | yes      | -                                                                  | Default Azurite connection string from documentation used for a local development.                                                                                                                                                                                                                               |
+| AZURE_STORAGE_LAYER_CONTAINER_NAME | yes      | layer-geo-data                                                     | Name of the Blob Container to which geojsons are saved in the Azurite.                                             |
+| MONGO_URI                          | yes      | mongodb://localhost:27917/oscar or mongodb://localhost:27017/oscar | URI with port on which the MongoDB is running. In the development environment it runs on port 27917 and in the testing environment on port 27017.                                                                                                                                                                |
+| UPLOAD_DATA_TYPES                  | no       | -                                                                  | Data sets to be uploaded into the database. There are two options: 'mapLayers' for the map view configuration and 'attributes' for application data.                                                                                                                                                             |
```

Comment:
Is it clear that you can use both in the same run?

Reply:
I fixed this.